script.js

```diff
@@ -0,0 +1,61 @@
+async function handleSubmit(e) {
+    e.preventDefault();
+  document.getElementById("result-head").innerHTML = "";
+  const result = document.getElementById("result");
+  result.innerHTML = "";
+
+  const profile = await getProfile();
+
+  if (profile != null) {
+    document
+      .getElementById("result-div")
+      .classList.add("container", "bg-white", "p-4");
+    document.getElementById("result-head").innerHTML = "Github Profile Details";
+
+    const li1 = document.createElement("li");
```

Comment:
I think it has some room for improvements for better UI

Reply:
oh ok